feature/platform/tests/optics_power_and_bias_current_test/optics_power_and_bias_current_test.go

```diff
@@ -143,7 +129,7 @@ func TestOpticsPowerUpdate(t *testing.T) {
 
 			component := gnmi.OC().Component(transceiverName)
 			if !gnmi.Lookup(t, dut, component.MfgName().State()).IsPresent() {
-				t.Skipf("component.MfgName().Lookup(t).IsPresent() for %q is false. skip it", transceiverName)
+				t.Fatalf("component.MfgName().Lookup(t).IsPresent() for %q is false", transceiverName)
```

Comment:
I would actually suggest to remove this whole `if` block, because It's not the focus of this test, and is not a must to have by the test (to validate the transceiver power).